DataFormats/Detectors/FIT/FT0/include/DataFormatsFT0/DigitsTemp.h

```diff
@@ -0,0 +1,90 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#ifndef ALICEO2_FIT_DIGIT_H
+#define ALICEO2_FIT_DIGIT_H
+
+#include "CommonDataFormat/InteractionRecord.h"
+#include "CommonDataFormat/TimeStamp.h"
+#include "DataFormatsFT0/Digit.h"
+#include "DataFormatsFT0/ChannelData.h"
+#include <iosfwd>
+#include "Rtypes.h"
+
+namespace o2
+{
+namespace ft0
+{
+
+/// \class Digit
+/// \brief FIT digit implementation
+using DigitBase = o2::dataformats::TimeStamp<double>;
+class DigitsTemp : public DigitBase
```

Comment:
could you add a sentence of what is "Temp" here? Is it some intermediate format?